DTOs/Ticket/CreateTicketRequestDto.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using System.ComponentModel.DataAnnotations;
+using GeekHub.DTOs.ApplicationUser;
+using GeekHub.Models;
+
+namespace GeekHub.DTOs.Ticket
+{
+    public class CreateTicketRequestDto
+    {
+        [Required] public string Name { get; set; }
+        public string Description { get; set; }
+
+        [Required]
+        [EnumDataType(typeof(TicketType))]
+        public TicketType TicketType { get; set; }
+
+        [Required]
+        [EnumDataType(typeof(TicketStatus))]
+        public TicketStatus TicketStatus { get; set; }
```

Comment:
Do we need to think about extensibility, or will this be easy to extend later (e.g. adding labels, links, attachments, etc...)

Reply:
The DTO would need to be updated then, but it shouldn't be too hard...